### ISSUE TITLE
A consultation over the results by your released code

### ISSUE THREAD

**Issue by Starboy-at-earth** [ZJULearning/resa#21]
Dear @comzyh :
         It seems that you have not prepared to release the inference code, at least in this REPO recently. Emmm, here is the thing, and your reply is very desired. 
         My  trouble is that the performance reported in your paper cannot be determined reproduced, successfully or not, by deploying this project.  I get the following results. Could you please tell me whether these are approximate to yours on the validation set of the TuSimple?

########################################
2021-06-21 10:20:51,067 - resa - INFO - [{"name": "Accuracy", "value": 0.9450447819314601, "order": "desc"}, {"name": "FP", "value": 0.07953510663791019, "order": "asc"}, {"name": "FN", "value": 0.08024802300503252, "order": "asc"}]
2021-06-21 10:20:51,399 - resa - INFO - Best metric: 0.9450447819314601
########################################

B.T.W., anyone with the same puzzle is also desired.

**Comment by Turoad** [comment]
Why you @"comzyh" ?
I don't know how you train. Please show more information.

**Comment by Turoad** [comment]
In https://github.com/ZJULearning/resa/issues/16, they can reproduce the result. I believe there is something wrong in your training setting. I close this issue first. If you have other questions, please raise again.
